python/src/trezorlib/models.py

```diff
@@ -96,7 +96,7 @@ class TrezorModel:
 TREZOR_T = T2T1
 TREZOR_R = T2B1
 TREZOR_SAFE3 = T2B1
-TREZOR_T3T1 = T2B1
+TREZOR_T3T1 = T3T1
```

Comment:
please completely drop this line.
alias "TREZOR T3T1" for just "T3T1" is not useful for anything

Reply:
Right, done in force-push with changed commit message